lib/skate/application.ex

```diff
@@ -14,6 +16,14 @@ defmodule Skate.Application do
       # {Skate.Worker, arg},
     ]
 
+    # Don't start the Gtfs gen server when testing, because it's slow to start
```

Comment:
It's slow to start because of the processing that's happening at startup. You probably won't need this if you do the `init` refactoring.